packages/web-client/app/utils/web3-strategies/layer1-chain.ts

```diff
@@ -303,6 +309,14 @@ export default abstract class Layer1ChainWeb3Strategy
     );
   }
 
+  async resumeRelayTokens(
+    txnHash: TransactionHash
+  ): Promise<TransactionReceipt> {
+    if (!this.web3) throw new Error('Cannot unlock tokens without web3');
```

Comment:
In which case can this happen - the absence of `this.web3`?

Reply:
Good question. I think it might be there to avoid a typescript "!"